docs/standard/data/xml/xslt-security-considerations.md

```diff
@@ -61,4 +61,4 @@ The XSLT language has a rich set of features that give you a great deal of power
 ## See Also  
  [XSLT Transformations](../../../../docs/standard/data/xml/xslt-transformations.md)   
  [Resolving External Resources During XSLT Processing](../../../../docs/standard/data/xml/resolving-external-resources-during-xslt-processing.md)   
- [Code Access Security](http://msdn.microsoft.com/en-us/23a20143-241d-4fe5-9d9f-3933fd594c03)
+ [Code Access Security](http://msdn.microsoft.com/23a20143-241d-4fe5-9d9f-3933fd594c03)
```

Comment:
same here: please change the CAS link to ~/docs/framework/misc/code-access-security.md